compiler/damlc/tests/daml-test-files/TextMap.daml

```diff
@@ -46,26 +46,26 @@ testNull = scenario do
 
 testEq = scenario do
   (TM.empty : TextMap Int) === (TM.empty : TextMap Int)
-  assert (not (TM.empty == (TM.fromList [("1", 1)])))
-  (TM.fromList [("1", 1), ("2", 2), ("3", 3)]) === (TM.fromList [("1", 1), ("2", 2), ("3", 3)])
-  assert (not ((TM.fromList [("1", 1), ("2", 2), ("3", 3)]) == (TM.fromList [("1", 2), ("2", 2)])))
-  assert (not ((TM.fromList [("1", 1), ("2", 2), ("3", 3)]) == (TM.fromList [("1", 2), ("2", 2), ("3", 4)])))
-  assert (not ((TM.fromList [("1", 1), ("2", 2), ("3", 3)]) == (TM.fromList [("1", 2), ("2", 2), ("4", 3)])))
+  assert (not (TM.empty == TM.fromList [("1", 1)]))
```

Comment:
The fact that our own tests have tons of redundant parentheses seems to confirm my comment that parentheses might be easier than remembering precedences :slightly_smiling_face:

Reply:
Less than I might have anticipated. I hadn't thought about it that way but thinking back, the advice when learning most new programming languages is when in doubt, use brackets to be sure.